components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/typing.scala

```diff
@@ -474,7 +474,7 @@ object typing {
     superTypeOfTypes(list.map(fromInstance))
   }
 
-  private def superTypeOfTypes(list: Iterable[TypingResult]) = {
+  def superTypeOfTypes(list: Iterable[TypingResult]) = {
```

Comment:
`private[typed]`, we want to expose the narrowest set of methods in the api. This is an api exposed for developers of components